### PR TITLE
[MARLIN-382] dashboard widget kpi creation dates

### PR DESCRIPTION
**Pull request by iseessel**

### CODE REVIEW

**Comment by iseessel** [comment]
@alexmaestrano @ouranos @yuritomanek @paulstuartparker 

Plz review.

File: src/app/components/mnoe-dashboards-list/mnoe-dashboards-list.coffee [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#
+# Mnoe Dashboard Templates list
```
Comment (iseessel) [inline src/app/components/mnoe-dashboards-list/mnoe-dashboards-list.coffee]: Outdated comment

File: src/app/components/mnoe-dashboards-list/mnoe-dashboards-list.coffee [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+#
+@App.component('mnoeDashboardsList', {
+  templateUrl: 'app/components/mnoe-dashboards-list/mnoe-dashboards-list.html',
+  bindings: {
+    userId: '<'
```
Comment (iseessel) [inline src/app/components/mnoe-dashboards-list/mnoe-dashboards-list.coffee]: outdated

File: src/app/components/mnoe-dashboards-list/mnoe-dashboards-list.coffee [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  },
+  controller: ($uibModal, toastr, MnoeUsers, $stateParams) ->
```
Comment (iseessel) [inline src/app/components/mnoe-dashboards-list/mnoe-dashboards-list.coffee]: not using $uibModal or toaster

File: src/app/views/user/user.html [hunk 4]
```diff
@@ -82,9 +82,12 @@ <h3 class="panel-title">{{'mnoe_admin_panel.dashboard.user.geo_information_title
     </div>
     <div class="bs-row">
       <mnoe-impersonate-button wrapper-class-names="col-sm-4 col-sm-offset-4" btn-class-names="btn btn-primary center-block" user="vm.user"></mnoe-impersonate-button>
-      <div class="col-sm-12 top-buffer-2">
+      <div class="col-sm-12 top-buffer-2 no-padding">
```
Comment (iseessel) [inline src/app/views/user/user.html]: no-padding not used

**Comment by iseessel** [comment]
screen shot 2018-10-17 at 5 11 24 pm

File: src/locales/en-AU.json [hunk 5]
```diff
@@ -223,6 +223,13 @@
   "mnoe_admin_panel.dashboard.product_markups.offers.modal.table_headers.final_prices": "Final Prices",
   "mnoe_admin_panel.dashboard.product_markups.offers.modal.usage_based": "usage based",
   "mnoe_admin_panel.dashboard.product_markups.offers.modal.one_off": "one off",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.title": "User dashboards",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.name": "Name",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.currency": "Currency",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.updated_at": "Updated at",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.created_at": "Created at",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.no_kpis": "This dashboard does not have any kpis.",
```
Comment (alexmaestrano) [inline src/locales/en-AU.json]: If customers see this, we could format this to "KPIs" instead of "kpis"

Comment (iseessel) [reply src/locales/en-AU.json]: 👍

Comment (iseessel) [reply src/locales/en-AU.json]: 👍

File: src/app/components/mnoe-dashboards-list/mnoe-dashboards-list.html [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+        <tbody ng-repeat="dashboard in $ctrl.dashboards.list">
+          <tr ng-click="$ctrl.toggleDashboard(dashboard)">
+            <td title="{{ dashboard.full_name }}">
+              <span>
+                <i class="fa" ng-class="{ 'fa-caret-right': !dashboard.opened, 'fa-caret-down': dashboard.opened }"></i>
+              </span>
+              {{ dashboard.name }}
+            </td>
+            <td title="{{ dashboard.full_name }}">{{ $ctrl.currency(dashboard) }}</td>
+            <td title="{{ dashboard.full_name }}">{{ dashboard.updated_at | date: "dd/MM/yyyy \'at\' h:mm a" }}</td>
```
Comment (adamaziz15) [inline src/app/components/mnoe-dashboards-list/mnoe-dashboards-list.html]: Might be good to consider using angular moment here, something like:
```
{{ dashboard.updated_at | amDateFormat: 'L' }}
```
this way the format will be based on locale
More info on the moment pipes here: https://github.com/urish/ngx-moment

Same for the rest of the dates

File: src/app/views/user/user.html [hunk 7]
```diff
@@ -85,6 +85,9 @@ <h3 class="panel-title">{{'mnoe_admin_panel.dashboard.user.geo_information_title
       <div class="col-sm-12 top-buffer-2">
         <mnoe-organizations-local-list list="vm.user.organizations" user="vm.user" view="all"></mnoe-organizations-local-list>
       </div>
+      <div class="col-sm-12 top-buffer-2">
+        <mnoe-dashboards-list ng-if="vm.isAdminDashboardViewingEnabled"></mnoe-organizations-local-list>
```
Comment (adamaziz15) [inline src/app/views/user/user.html]: looks like a little copy and paste :trollface: - opening and closing tags don't match

Comment (iseessel) [reply src/app/views/user/user.html]: I would never copy and paste code.

**Pull request merged**